tasks/install-r.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: install | r-version (download)
+  get_url:
+    url: "https://cdn.rstudio.com/r/ubuntu-2004/pkgs/r-{{ item }}_1_amd64.deb" 
+    dest: /tmp/r-{{ item }}_1_amd64.deb
+  loop: "{{ r_versions }}"
+  tags:
+    - r-version-install
+
+- name: install | r-version
+  command: gdebi --n /tmp/r-{{ item }}_1_amd64.deb
```

Comment:
Do we need `gdebi` to install `.deb` package? Ansible's builtin apt module can install `.deb` as well: https://github.com/Appsilon/ansible-rstudio-connect/blob/main/tasks/install.yml#L43

Reply:
Tried with apt, worked and pushed with new commit.